src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -180,29 +189,34 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
       source?: string;
     };
   } = {
+    concealedSamples: {
+      transform: this.countPerSecond,
+    },
+    totalSamplesReceived: {
+      transform: this.countPerSecond,
+    },
```

Comment:
For my understanding, what happens if we don't explicitly specify the type here?

Reply:
We just keep it locally and not sent them to Tincan. The `type` is actually the protobuf.
https://github.com/aws/amazon-chime-sdk-js/blob/e67ada29b03207aec0e9b68f245963dcfb0bab7b/src/statscollector/DefaultStatsCollector.ts#L308